Makefile

```diff
@@ -334,6 +337,7 @@ install: $(build_depsbindir)/stringreplace $(BUILDROOT)/doc/_build/html
 	done
 
 	$(INSTALL_M) $(build_bindir)/julia* $(DESTDIR)$(bindir)/
+	$(INSTALL_M) $(build_bindir)/embedding* $(DESTDIR)$(bindir)/
```

Comment:
And because of the toolchain requirement, I don't think we'd include it in the default list in `test/choosetests.jl`, but make it one you have to ask for - like `libgit2-online` and `pkg`, but for different reasons.